.github/CODEOWNERS

```diff
@@ -1,2 +1,8 @@
-# These users will be requested for review when someone opens a pull request.
-*       @luongvo @sleepylee @hoangnguyen92dn @lydiasama @Tuubz @minhnimble @manh-t @AVI5HEK @Wadeewee @chornerman @doannimble @ryan-conway @kaungkhantsoe
+# Team Lead
+* @Tuubz
+
+# Team Members
+* @AVI5HEK @chornerman @doannimble @hoangnguyen92dn @kaungkhantsoe @luongvo @lydiasama @manh-t @minhnimble @ryan-conway @sleepylee @Wadeewee 
+
+# Engineering Leads
+CODEOWNERS @nimblehq/engineering-leads
```

Comment:
Does `CODEOWNERS` do anything special? Just wondering 😁

Reply:
@Tuubz I can't find an official doc but can guess it will tag our ELs into PRs from this team https://github.com/orgs/nimblehq/teams/engineering-leads

Reply:
@olivierobert Do we have any official documentation about `CODEOWNERS` keyword usages?

Reply:
Not yet. It remains to be added on Compass 🙇

Reply:
Here is [the doc from Github](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).

Reply:
@Tuubz Ahh I got it, we can notify a team or member when having changes in this actual `CODEOWNERS` file, `CODEOWNERS` is not a keyword but this file name.

<img width="594" alt="image" src="https://user-images.githubusercontent.com/16315358/160573421-882c5949-bf49-425c-9e48-a4bafb138888.png">

Reply:
Aaaha, makes sense. I also thought it was used as a keyword, thanks for the clarification ⭐